gatsby-config.js

```diff
@@ -11,6 +11,9 @@ const path = require('path');
 
 require('dotenv').config();
 envalid.cleanEnv(process.env, { ABSTRACT_TOKEN: envalid.str() });
+envalid.cleanEnv(process.env, { FIGMA_TOKEN: envalid.str() });
```

Comment:
This can simply be added to the object above...

`{ ABSTRACT_TOKEN: envalid.str(), FIGMA_TOKEN: envalid.str() }`